ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -346,7 +350,24 @@ public void postGetHeadDeleteTest()
     userMetadata.put(RestUtils.Headers.USER_META_DATA_HEADER_PREFIX + "key2", "value2");
     RestUtilsTest.setUserMetadataHeaders(headers, userMetadata);
     String blobId = postBlobAndVerify(headers, content);
-    getBlobAndVerify(blobId, headers, content);
+
+    getBlobAndVerify(blobId, null, headers, content);
+
+    ByteRange range = ByteRange.fromStartOffset(ThreadLocalRandom.current().nextLong(CONTENT_LENGTH));
+    getBlobAndVerify(blobId, range, headers, content);
+
+    range = ByteRange.fromLastNBytes(ThreadLocalRandom.current().nextLong(CONTENT_LENGTH + 1));
+    headers.put(RestUtils.Headers.CONTENT_RANGE,
```

Comment:
Perhaps you can remove this (like in the previous case).

Remove for the next case also.